app/models/hydrus/generic_object.rb

```diff
@@ -312,7 +312,7 @@ def self.dor_registration_params(user_string, obj_typ, apo_pid)
 
       params[:administrative][:partOfProject] = 'Hydrus' unless obj_typ == 'adminPolicy'
       params[:identification] = { sourceId: "Hydrus:#{obj_typ}-#{user_string}-#{HyTime.now_datetime_full}" } if obj_typ == 'item'
-      params[:access] = {} if obj_typ == 'collection'
+      params[:access] = { 'access': 'world' } if obj_typ == 'collection'
```

Comment:
This is very minor, but can you change `'access':` to `access:` so it looks like a garden-variety symbolized hash?

Reply:
it was choking but maybe I wrote my test wrong - will take a look.

Reply:
suggestion taken.